integration_tests/e2eReferDisabled/refer.cy.ts

```diff
@@ -68,17 +68,20 @@ context('Refer', () => {
     notFoundPage.shouldContain404H2()
   })
 
-  it("Doesn't show the the in-progress referral task list", () => {
+  it("Doesn't show the in-progress referral task list", () => {
     cy.signIn()
 
+    const referral = referralFactory.build()
     const course = courseFactory.build()
-    const courseOffering = courseOfferingFactory.build()
+    const courseOffering = courseOfferingFactory.build({ id: referral.offeringId })
     const prison = prisonFactory.build({ prisonId: courseOffering.organisationId })
-    const referral = referralFactory.build()
+    const prisoner = prisonerFactory.build({ prisonerNumber: referral.prisonNumber })
 
     cy.task('stubCourseByOffering', { course, courseOfferingId: courseOffering.id })
     cy.task('stubCourseOffering', { courseId: course.id, courseOffering })
     cy.task('stubPrison', prison)
+    cy.task('stubPrisoner', prisoner)
+    cy.task('stubReferral', referral)
```

Comment:
Explain this in the commit - we now hit more endpoints in the related controller action so we need to stub them here to avoid false negatives